packages/marko-web-theme-monorail/scss/components/_spec-guide.scss

```diff
@@ -0,0 +1,37 @@
+$common-spec-guide-container-max-width: 1400px;
+
+// Require Bootstrap tables and input groups
+@import "../../node_modules/bootstrap/scss/tables";
+@import "../../node_modules/bootstrap/scss/input-group";
```

Comment:
This requires adding bootstrap@`4.3.1` as a dependency in the theme `package.json`